ui/src/components/Header/Header.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './Header.css';
+
+/* Profile photo link  be passed in from the Page component */
+const Header = (props) => {
+    return(
+        <div className="header">
+            <img className="profile-pic" src={require(`../../img/${props.imgLink}`)} alt="Profile Pic"/>
```

Comment:
```js
  import './Header.css';
  import sampleProfilePic from '../../img/profile.png';
```
The image link can be changed later when we use a specific user's personal profile image.
(there's also the problem of us not being supposed to use 'require' here)